index.d.ts

```diff
@@ -58,18 +66,19 @@ declare namespace Moleculer {
 		static createFromPayload(broker: ServiceBroker, payload: object): Context;
 	}
 
-	interface ServiceActionSchema {
-		cache?: boolean;
-		params?: object;
-		handler: (ctx: Context) => Promise<any>;
-	}
-
 	interface ServiceSettingSchema {
 		port?: number | string;
-		routes?: Route[];
+		routes?: RouteSchema[];
```

Comment:
Are port and routes not specific to the moleculer-web module? If so they should be in that project as an extension of this interface.

Reply:
Yes, It's true. I introduced this mistake, I thought `ServiceSettingSchema` has these two properties in `moleculer` by default. I'm sorry.

Reply:
Please remove them.